packages/create-svelte/templates/default/src/lib/db.ts

```diff
@@ -0,0 +1,89 @@
+import fetch from 'node-fetch';
```

Comment:
I'm still a bit unsure about hitting a real api. Couldn't we do a mock here instead and write to local storage? If everyone else uses the db interface we can showcase these things just the same.

Reply:
You can't make an app that works without JS if you rely on localStorage